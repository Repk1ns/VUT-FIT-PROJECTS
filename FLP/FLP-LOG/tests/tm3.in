S   A R
A b A R
A c A R
A x A R
A a B x
B b B L
B c B L
B x B L
B   C R
C a C R
C c C R
C x C R
C b D x
D a D L
D c D L
D b D L
D x D L
D   E R
E b E R
E c E R
E x E R
E a G x
G b G R
G c G R
G x G R
G a H x
H a H L
H c H L
H b H L
H x H L
H   C R
C   F   
 aabbacacabaabcaa