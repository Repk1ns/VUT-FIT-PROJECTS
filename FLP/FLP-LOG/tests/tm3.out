S aabbacacabaabcaa
 Aaabbacacabaabcaa
 Bxabbacacabaabcaa
B xabbacacabaabcaa
 Cxabbacacabaabcaa
 xCabbacacabaabcaa
 xaCbbacacabaabcaa
 xaDxbacacabaabcaa
 xDaxbacacabaabcaa
 Dxaxbacacabaabcaa
D xaxbacacabaabcaa
 Exaxbacacabaabcaa
 xEaxbacacabaabcaa
 xGxxbacacabaabcaa
 xxGxbacacabaabcaa
 xxxGbacacabaabcaa
 xxxbGacacabaabcaa
 xxxbHxcacabaabcaa
 xxxHbxcacabaabcaa
 xxHxbxcacabaabcaa
 xHxxbxcacabaabcaa
 Hxxxbxcacabaabcaa
H xxxbxcacabaabcaa
 Cxxxbxcacabaabcaa
 xCxxbxcacabaabcaa
 xxCxbxcacabaabcaa
 xxxCbxcacabaabcaa
 xxxDxxcacabaabcaa
 xxDxxxcacabaabcaa
 xDxxxxcacabaabcaa
 Dxxxxxcacabaabcaa
D xxxxxcacabaabcaa
 Exxxxxcacabaabcaa
 xExxxxcacabaabcaa
 xxExxxcacabaabcaa
 xxxExxcacabaabcaa
 xxxxExcacabaabcaa
 xxxxxEcacabaabcaa
 xxxxxcEacabaabcaa
 xxxxxcGxcabaabcaa
 xxxxxcxGcabaabcaa
 xxxxxcxcGabaabcaa
 xxxxxcxcHxbaabcaa
 xxxxxcxHcxbaabcaa
 xxxxxcHxcxbaabcaa
 xxxxxHcxcxbaabcaa
 xxxxHxcxcxbaabcaa
 xxxHxxcxcxbaabcaa
 xxHxxxcxcxbaabcaa
 xHxxxxcxcxbaabcaa
 Hxxxxxcxcxbaabcaa
H xxxxxcxcxbaabcaa
 Cxxxxxcxcxbaabcaa
 xCxxxxcxcxbaabcaa
 xxCxxxcxcxbaabcaa
 xxxCxxcxcxbaabcaa
 xxxxCxcxcxbaabcaa
 xxxxxCcxcxbaabcaa
 xxxxxcCxcxbaabcaa
 xxxxxcxCcxbaabcaa
 xxxxxcxcCxbaabcaa
 xxxxxcxcxCbaabcaa
 xxxxxcxcxDxaabcaa
 xxxxxcxcDxxaabcaa
 xxxxxcxDcxxaabcaa
 xxxxxcDxcxxaabcaa
 xxxxxDcxcxxaabcaa
 xxxxDxcxcxxaabcaa
 xxxDxxcxcxxaabcaa
 xxDxxxcxcxxaabcaa
 xDxxxxcxcxxaabcaa
 Dxxxxxcxcxxaabcaa
D xxxxxcxcxxaabcaa
 Exxxxxcxcxxaabcaa
 xExxxxcxcxxaabcaa
 xxExxxcxcxxaabcaa
 xxxExxcxcxxaabcaa
 xxxxExcxcxxaabcaa
 xxxxxEcxcxxaabcaa
 xxxxxcExcxxaabcaa
 xxxxxcxEcxxaabcaa
 xxxxxcxcExxaabcaa
 xxxxxcxcxExaabcaa
 xxxxxcxcxxEaabcaa
 xxxxxcxcxxGxabcaa
 xxxxxcxcxxxGabcaa
 xxxxxcxcxxxHxbcaa
 xxxxxcxcxxHxxbcaa
 xxxxxcxcxHxxxbcaa
 xxxxxcxcHxxxxbcaa
 xxxxxcxHcxxxxbcaa
 xxxxxcHxcxxxxbcaa
 xxxxxHcxcxxxxbcaa
 xxxxHxcxcxxxxbcaa
 xxxHxxcxcxxxxbcaa
 xxHxxxcxcxxxxbcaa
 xHxxxxcxcxxxxbcaa
 Hxxxxxcxcxxxxbcaa
H xxxxxcxcxxxxbcaa
 Cxxxxxcxcxxxxbcaa
 xCxxxxcxcxxxxbcaa
 xxCxxxcxcxxxxbcaa
 xxxCxxcxcxxxxbcaa
 xxxxCxcxcxxxxbcaa
 xxxxxCcxcxxxxbcaa
 xxxxxcCxcxxxxbcaa
 xxxxxcxCcxxxxbcaa
 xxxxxcxcCxxxxbcaa
 xxxxxcxcxCxxxbcaa
 xxxxxcxcxxCxxbcaa
 xxxxxcxcxxxCxbcaa
 xxxxxcxcxxxxCbcaa
 xxxxxcxcxxxxDxcaa
 xxxxxcxcxxxDxxcaa
 xxxxxcxcxxDxxxcaa
 xxxxxcxcxDxxxxcaa
 xxxxxcxcDxxxxxcaa
 xxxxxcxDcxxxxxcaa
 xxxxxcDxcxxxxxcaa
 xxxxxDcxcxxxxxcaa
 xxxxDxcxcxxxxxcaa
 xxxDxxcxcxxxxxcaa
 xxDxxxcxcxxxxxcaa
 xDxxxxcxcxxxxxcaa
 Dxxxxxcxcxxxxxcaa
D xxxxxcxcxxxxxcaa
 Exxxxxcxcxxxxxcaa
 xExxxxcxcxxxxxcaa
 xxExxxcxcxxxxxcaa
 xxxExxcxcxxxxxcaa
 xxxxExcxcxxxxxcaa
 xxxxxEcxcxxxxxcaa
 xxxxxcExcxxxxxcaa
 xxxxxcxEcxxxxxcaa
 xxxxxcxcExxxxxcaa
 xxxxxcxcxExxxxcaa
 xxxxxcxcxxExxxcaa
 xxxxxcxcxxxExxcaa
 xxxxxcxcxxxxExcaa
 xxxxxcxcxxxxxEcaa
 xxxxxcxcxxxxxcEaa
 xxxxxcxcxxxxxcGxa
 xxxxxcxcxxxxxcxGa
 xxxxxcxcxxxxxcxHx
 xxxxxcxcxxxxxcHxx
 xxxxxcxcxxxxxHcxx
 xxxxxcxcxxxxHxcxx
 xxxxxcxcxxxHxxcxx
 xxxxxcxcxxHxxxcxx
 xxxxxcxcxHxxxxcxx
 xxxxxcxcHxxxxxcxx
 xxxxxcxHcxxxxxcxx
 xxxxxcHxcxxxxxcxx
 xxxxxHcxcxxxxxcxx
 xxxxHxcxcxxxxxcxx
 xxxHxxcxcxxxxxcxx
 xxHxxxcxcxxxxxcxx
 xHxxxxcxcxxxxxcxx
 Hxxxxxcxcxxxxxcxx
H xxxxxcxcxxxxxcxx
 Cxxxxxcxcxxxxxcxx
 xCxxxxcxcxxxxxcxx
 xxCxxxcxcxxxxxcxx
 xxxCxxcxcxxxxxcxx
 xxxxCxcxcxxxxxcxx
 xxxxxCcxcxxxxxcxx
 xxxxxcCxcxxxxxcxx
 xxxxxcxCcxxxxxcxx
 xxxxxcxcCxxxxxcxx
 xxxxxcxcxCxxxxcxx
 xxxxxcxcxxCxxxcxx
 xxxxxcxcxxxCxxcxx
 xxxxxcxcxxxxCxcxx
 xxxxxcxcxxxxxCcxx
 xxxxxcxcxxxxxcCxx
 xxxxxcxcxxxxxcxCx
 xxxxxcxcxxxxxcxxC
 xxxxxcxcxxxxxcxxF
